include/fmt/format.cc

```diff
@@ -280,9 +280,16 @@ const uint64_t internal::basic_data<T>::POWERS_OF_10_64[] = {
 
 FMT_FUNC internal::utf8_to_utf16::utf8_to_utf16(string_view s) {
   static const char ERROR_MSG[] = "cannot convert string from UTF-8 to UTF-16";
-  if (s.size() > INT_MAX)
+  if (s.size() > INT_MAX || s.data() == nullptr)
     FMT_THROW(windows_error(ERROR_INVALID_PARAMETER, ERROR_MSG));
   int s_size = static_cast<int>(s.size());
+  if (s_size == 0) // MultiByteToWideChar does not support zero length, handle separately
+  {
+      buffer_.resize(1);
+      buffer_[0] = 0;
+      return;
+  }
```

Comment:
Please follow the Google style guide (2-space indent and open brace on the same line:

```c++
if (s_size == 0) { // MultiByteToWideChar does not support zero length, handle separately
  buffer_.resize(1);
  ...
```

Same below.

Reply:
Sure. Will fix.
Shall I submit a new (v2) pull-request, to reduce the fix commits clutter?

Reply:
No, they can be squashed together before merging.